pkg/machines/components/create-vm-dialog/Password.css

```diff
@@ -0,0 +1,23 @@
+/* user config - password strength */
+.password-strength-meter {
+  height: 5px;
+  margin-top: 5px;
+  margin-bottom: 0;
+}
+.password-strength-meter div {
+  width: 25%;
+  border-right: 3px solid #fff;
+  background-color: transparent;
+}
+.password-strength-meter.weak div:first-child {
+  background-color: #C00;
```

Comment:
could you please use some variables we have defined? It makes it much easier to have control over the style of cockpit as a whole.

Reply:
I don't see this C00 defined anywhere though, do be honest I copied that password component code from composer, that's why you saw some inconstancies already.

Reply:
`--color-red-1`?

Reply:
Oh thanks